packages/core/src/Stores/traders-hub-store.js

```diff
@@ -331,6 +334,10 @@ export default class TradersHubStore extends BaseStore {
         this.is_regulators_compare_modal_visible = !this.is_regulators_compare_modal_visible;
     }
 
+    setMT5NotificationModal(is_visible) {
+        this.is_mt5_notificaiton_modal_visible = is_visible;
+    }
```

Comment:
Usually, we have these boolean states for modals in the UI store, Not sure if we should move this there or not 🤔 You can ignore this comment 🙇🏻

Reply:
Ignore it for now